public/app/features/alerting/unified/components/contact-points/ContactPoint.tsx

```diff
@@ -176,6 +170,11 @@ export const ContactPointReceiverSummary = ({ receivers, limit }: ContactPointRe
   return (
     <Stack direction="column" gap={0}>
       <Stack direction="row" alignItems="center" gap={1}>
+        {integrationsShown.length === 0 && (
+          <MetaText color="warning" icon="exclamation-circle">
```

Comment:
nit: for warnings we tend to use `exclamation-triangle`, though I do realize this was just code that was moved ;)